localization/yabloc/yabloc_pose_initializer/src/camera/semantic_segmentation.cpp

```diff
@@ -53,7 +53,8 @@ cv::Mat SemanticSegmentation::convert_blob_to_image(const cv::Mat & blob)
   const int channels = blob.size[1];
   const int height = blob.size[2];
   const int width = blob.size[3];
-  cv::Mat image = cv::Mat(height, width, CV_32FC4);
+  cv::Mat image = cv::Mat(height, width, CV_32FC4);  // NOLINT
+                                                     // suppress hicpp-signed-bitwise from OpenCV
```

Comment:
May be we can revert the comment and remain the error showing.
This might get solved by changing the clang-tidy configuration. Currently, it is printing the `error: '_Atomic' is a C11 extension` anyway.

Reply:
I see. Thank you for the explanation! 
Since we don't know when this bug(?) will be fixed, I think it's fine to leave it as is. :ok_hand: 

Here is the issue addressing thread:  https://github.com/orgs/autowarefoundation/discussions/4876#discussioncomment-9900497